intercom_flutter/ios/intercom_flutter.podspec

```diff
@@ -3,7 +3,7 @@
 #
 Pod::Spec.new do |s|
   s.name             = 'intercom_flutter'
-  s.version          = '4.0.0'
+  s.version          = '5.0.0'
```

Comment:
Why it is required to update?

Reply:
Fixes an issue when building in CodeMagic, not sure why

Reply:
Does that mean this version should be updated every time as the new version of the package is released?

Reply:
@Zazo032 @v3rm0n should we merge this change? In fact, I didn't understand why it is required?

Reply:
It's not required, for example, you can check the `battery` [plugin](https://github.com/flutter/plugins/blob/master/packages/battery/battery/ios/battery.podspec), pub version is 2.0.3 and podspec file still has 0.0.1

Reply:
I did more testing and indeed this was not related to the error... but it's nice to have it with the same version...

Reply:
@Zazo032 what do think then? Should we update the version?

Reply:
We can merge the change, but it won't have any functional change

Reply:
👍